--- a/3rdparty/bgfx/src/renderer_gl.h	2021-02-23 16:02:38.000000000 +0100
+++ b/3rdparty/bgfx/src/renderer_gl.h	2021-06-19 22:30:23.266701378 +0200
@@ -21,10 +21,7 @@
 			) )
 
 #define BGFX_USE_WGL (BGFX_CONFIG_RENDERER_OPENGL && BX_PLATFORM_WINDOWS)
-#define BGFX_USE_GLX (BGFX_CONFIG_RENDERER_OPENGL && (0 \
-			|| BX_PLATFORM_BSD                          \
-			|| BX_PLATFORM_LINUX                        \
-			) )
+#define BGFX_USE_GLX 0
 
 #define BGFX_USE_GL_DYNAMIC_LIB (0 \
 			|| BX_PLATFORM_BSD     \
